app/helpers/application_helper.rb

```diff
@@ -0,0 +1,11 @@
+module ApplicationHelper
+	# Returns the full title on a per-page basis.
+	def full_title(page_title='')
+		base_title="Ruby on Rails Tutorial Sample App"
```

Comment:
I18n cho text `"Ruby on Rails Tutorial Sample App"` 
em tìm heieru về I18n và apply cho toàn bộ text dùng để hiển thị cho phía end user